wiki/Tournaments/CMC/2020/en.md

```diff
@@ -0,0 +1,388 @@
+---
+tags:
+  - CMC
+  - CMC 2020
+---
+
+# Cemara Mania Cup \#1 (2020)
+
+![CMC 2020 logo](img/logo.jpg)
+
+The **Cemara Mania Cup \#1 (2020)** (***CMC 2020***) was a double-elimination 2v2 osu!mania 4K tournament hosted by ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949) and co-hosted by ![][flag_ID] [LLENN-](https://osu.ppy.sh/users/7428653). The tournament was open to all osu!mania players from Indonesia with a [global osu!mania 4K rank](https://osu.ppy.sh/rankings/mania/performance?variant=4k) of #4000 or lower. It was the 1st iteration of the Cemara Mania Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2020-10-06/2020-11-01 |
+| Screening phase | 2020-11-02/2020-11-08 |
+| Qualifiers | 2020-11-09/2020-11-15 |
+| Group Stage | 2020-11-16/2020-11-22 |
+| Quarterfinals | 2020-11-23/2020-11-29 |
+| Semifinals | 2020-11-30/2020-12-06 |
+| Finals week 1 | 2020-12-07/2020-12-13 |
+| Finals week 2 | 2020-12-14/2020-12-20 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 3 months of osu!supporter tag for each team member + 100% of the raised prize pool |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag for each team member |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag for each team member |
+| *4th place* | Steam Wallet voucher worth approx. IDR 50.000,- for each team member |
```

Comment:
```suggestion
| *4th place* | Steam Wallet voucher worth approx. IDR 50.000 for each team member |
```

Reply:
i think i'll leave this as is since on [the ICC wiki article](http://osu.ppy.sh/wiki/en/Tournaments/ICC/2020) (which was also handled by me back then) the prize money has been specified like that

![image](https://user-images.githubusercontent.com/36564236/103252426-74a4e480-49af-11eb-96ed-5ba629a36531.png)